src/main/java/seedu/address/model/person/Email.java

```diff
@@ -64,4 +64,15 @@ public int hashCode() {
         return value.hashCode();
     }
 
+    /**
+     * Returns true if it contains same email
+     *
+     */
+    public boolean isSameEmail(Person person) {
```

Comment:
isSameEmail should compare between Emails, not between Email and Person

Reply:
while i know that is right, the implementation was because i had to use stream(). This means that i had no choice but to compare the email of the person with the email. I couldnt automatically extract the email before passing in the person. Perhaps we can talk through it in class?

Reply:
on second thought, i have completely removed it from email. I tried another easy method and it works